### PR TITLE
Add HTTP stats

### PR DESCRIPTION
**Pull request by mtparet**
Description and documentation are missing for now.

cf https://github.com/minio/minio/pull/5299 and https://github.com/minio/minio/pull/4049

### CODE REVIEW

File: minio_exporter.go [hunk 1]
```diff
@@ -171,6 +172,197 @@ func collectServerStats(e *MinioExporter, ch chan<- prometheus.Metric) {
...
+		ch <- prometheus.MustNewConstMetric(
+			prometheus.NewDesc(
+				prometheus.BuildFQName(namespace, "http", "success_count_heads"),
+				"Minio total output bytes received",
+				[]string{"minio_host"},
+				nil),
+			prometheus.GaugeValue,
+			float64(httpStats.SuccessHEADStats.Count), host)
+
+	    g, _ := strconv.ParseFloat(httpStats.SuccessHEADStats.AvgDuration, 64)
```
Comment (mtparet) [inline minio_exporter.go]: I'm not sure how to use this metric which is returned as a string https://github.com/minio/minio/blob/83abad0b3732da4bff851ef9c8a1bbabc9802ca2/cmd/http-stats.go#L100

**Comment by joe-pll** [comment]
Hi, thanks for the PR. I'll merge it as soon as it will be merged in madmin

**Comment by mtparet** [comment]
@joe-pll the PR has been merged in madmin and released :)

**Comment by joe-pll** [comment]
Hi @mtparet thanks a bunch for the PR. I merged it on the branch 'http-stats' and made a few changes to adapt it with the fixes I added on master. I will merge it on master if you do not have anything to add to this branch.

**Comment by mtparet** [comment]
Thank you @joe-pll ! It's good for me.

**Pull request merged**